trollflow2/plugins/__init__.py

```diff
@@ -257,11 +258,13 @@ def save_datasets(job):
     This is useful when other processes are waiting for the file to be present
     to start their work, but would crash on incomplete files.
 
-    If, in addition, the ``use_tmp_dir`` option is provided in the product
-    list, then the temporary file will be created in this directory without
+    If the ``staging_zone`` option is provided in the product
+    list, then the file will be created in this directory first, without
     changing the filename.  This is useful for writers which write the filename
     to the headers, such as the SatPy ninjotiff and ninjogeotiff writers.  The
-    value of ``use_tmp_dir`` has no effect if ``use_tmp_file`` is False.
+    ``staging_zone`` directory must be on the same filesystem as
+    ``output_dir``.  It is recommended to set ``use_tmp_file`` to `False` when
+    using a ``staging_zone`` directory.
```

Comment:
why?

Reply:
Because otherwise there's little point in using `staging_zone`.  The whole point is such that the filename written to the headers by the ninjo(geo)tiff writers is meaningful, which it isn't if `use_tmp_file` is False.  Edited documentation for clarifying this.